GooglePlayInstant/Editor/PlayInstantAssetBundleUrlVerifierWindow.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// Window that verifies AssetBundles from given URLs.
+    /// </summary>
+    public class PlayInstantAssetBundleUrlVerifierWindow : EditorWindow
```

Comment:
Let's simplify the name to something like AssetBundleVerifierWindow.

I know I used that "PlayInstant" prefix a lot in class names, but it felt necessary for a class like PlayInstantBuilder to not just name it Builder. In your case the name is already pretty specific.